nodecg-io-serial/extension/SerialClient.ts

```diff
@@ -0,0 +1,92 @@
+import { ServiceClient } from "nodecg-io-core/extension/types";
+import { success, error, Result } from "nodecg-io-core/extension/utils/result";
+import { ReadLine } from "readline";
+import SerialPort = require("serialport"); // This is neccesary, because serialport only likes require!
+// This is neccesary because serialport does not work with import...
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const Readline = require("@serialport/parser-readline");
+
+export interface DeviceInfo {
+    port: string;
+    manucaturer: string;
+    serialNumber: string;
+    pnpId: string;
+}
+
+interface Protocoll {
```

Comment:
Small typing here :wink: (also applies to the rest) 
```suggestion
interface Protocol {
```

Reply:
Thanks, will be fixed